### ISSUE TITLE
3-O-methyldopa (3-OMD) elevated

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11540]
**Preferred term label:**
 3-O-methyldopa (3-OMD) elevated (may need separate terms for CSF or blood)

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
For CSF, CSF neurotransmitter studies: https://www.mayocliniclabs.com/test-catalog/Overview/91688

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
aromatic L-amino acid decarboxylase deficiency https://omim.org/entry/608643 describes elevation in blood

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001471
- label: Elevated circulating 3-O-methyldopa concentration
- parents: [Abnormal circulating monocarboxylic acid concentration (HP:0010996)](https://hpo.jax.org/app/browse/term/HP:0010996)
- pmid: PMID:28100251
- def: The concentration of 3-O-methyldopa in the blood circulation is above the upper limit of normal.
- comment: 3-O-methyldopa (3-OMD) is produced by the methylation of L-DOPA by catechol-O-methyltransferase. 3-OMD may be elevated in the plasma and cerebrospinal fluid of Parkinson's disease patients taking L-DOPA and it is also observed to be elevated in individuals with Aromatic L-amino acid decarboxylase deficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11540
- synonyms: Elevated circulating 3-OMD concentration(NONE)|3-O-methyldopa elevated(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001472
- label: Elevated CSF 3-O-methyldopa concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: PMID:28100251
- def: The concentration of 3-O-methyldopa in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 3-O-methyldopa (3-OMD) is produced by the methylation of L-DOPA by catechol-O-methyltransferase. 3-OMD may be elevated in the plasma and cerebrospinal fluid of Parkinson's disease patients taking L-DOPA and it is also observed to be elevated in individuals with Aromatic L-amino acid decarboxylase deficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11540
- synonyms:
